src/renderer/component/common/form-components/form-field.jsx

```diff
@@ -24,6 +24,7 @@ type Props = {
   stretch?: boolean,
   affixClass?: string, // class applied to prefix/postfix label
   useToggle?: boolean,
+  firstInList?: boolean, // at the top of a list, no padding top
```

Comment:
I'm not super happy with this but currently there is a padding-top on all input elements. I plan to remove that because it's caused a lot of issues with input elelements that are in non-standard places (like this autoplay toggle).